src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java

```diff
@@ -580,6 +600,24 @@ public void close() throws IOException {
         eventLoopGroup.shutdownGracefully();
     }
 
+    /**
+     * Configure connection timeout in milliseconds
+     */
+    public NettyDockerCmdExecFactory withConnectTimeout(Integer connectTimeout) {
```

Comment:
I'm not sure if we should do that. Consider a new `DockerCmdExecFactory` implementation where this can't be configured...